explainers/signed-exchange-subresource-subtitution-explainer.md

```diff
@@ -0,0 +1,455 @@
+          rel="alternate";
+          type="application/signed-exchange;v=b3";
+          variants-05="accept;image/jpeg;image/webp";
+          variant-key-05="image/webp";
+          anchor="https://publisher.example/image";
+    ```
+- In **inner** response header of article.html.sxg:
+    ```
+    Link: <https://publisher.example/image>;
+          rel="allowed-alt-sxg";
+          variants-05="accept;image/jpeg;image/webp";
+          variant-key-05="image/jpeg";
+          header-integrity="sha256-AAA"
+    Link: <https://publisher.example/image>;
+          rel="allowed-alt-sxg";
+          variants-05="accept;image/jpeg;image/webp";
+          variant-key-05="image/webp";
+          header-integrity="sha256-BBB"
+    Link: <https://publisher.example/image>; rel=preload; as=image;
+    ```
+If a UA supports WebP, the UA should prefetch **image_webp.sxg** which content
+is WebP format. Otherwise the UA should prefetch **image_jpeg.sxg** which
+content is JPEG format.
+
+## Security and Privacy Considerations
+- The publishers can know whether the referrer page has prefetched the signed
+exchange subresources or not by checking the resource timing information. But
+this only exposes 1 bit information (= yes or no) because UAs can use the cached
+signed exchange only if the required signed exchanges are all available.
+- The UA can use the prefetched signed exchange subresources, only when they
+were prefetched in the referrer page. This is intended to avoid leaking the
+prefetching state to succeeding pages.
+- The UA check the header integrity value, so the distributor of the subresource
+signed exchange can’t inject arbitrary resources to the publisher’s page. This
+prevents distributors from sending tracking IDs to the publisher’s page.
+- The UA need to check both
+[the request URL](https://wicg.github.io/webpackage/loading.html#exchange-request-url)
+and the header integrity value of the signed exchange to avoid
+[Origin Laundering](https://hillbrad.github.io/sri-addressable-caching/sri-addressable-caching.html#origin_laundering)
+attack. The URL is not in signedHeaders, so the header integrity value can’t
+check the URL is correct.
+- The UA must fetch the signed exchange subresource
+(https://cdn.feed.example/cdn.publisher.example/lib.js.sxg) while processing the
+prefetch link element  (`<link rel="prefetch"
+href="https://cdn.feed.example/cdn.publisher.example/lib.js.sxg" as="script">`)
+even if there is the original subresource (http://cdn.publisher.example/lib.js)
+in the HTTPCache. Otherwise it leaks the state of  publisher’s site to the
+distributor of the signed exchange.
+- The UA must fetch the alternate signed exchange subresource (lib.js.sxg) even
+if there is the original subresource (lib.js) in the HTTPCache. Otherwise it
+leaks the state of publisher’s site to the distributor of the signed exchange.
+- To prevent tracking (user ID transfer), if the aggregator failed to prefetch a
```

Comment:
This is very similar to the first bullet in this list. Put them next to each other.